operators/online_video_reasoner/tests/test_media.py

```
@@ -0,0 +1,195 @@
+    assert process.reaped
+
+
+class _StalledProcess:
+    def __init__(self):
+        self.returncode = None
+        self.killed = False
+        self.reaped = False
+        self.communicate_calls = []
+        self.communication_started = threading.Event()
+
+    def communicate(self, input=None, timeout=None):
```

Ruff (0.15.21)

[error] Function argument `input` is shadowing a Python builtin

(A002)